lightning/src/util/config.rs

```diff
@@ -149,7 +148,28 @@ pub struct ChannelHandshakeConfig {
 	///                as 1000 sats instead, which is a safe implementation-specific lower bound.
 	/// Maximum value: 1,000,000, any values larger than 1 Million will be treated as 1 Million (or 100%)
 	///                instead, although channel negotiations will fail in that case.
-	pub their_channel_reserve_proportional_millionths: u32
+	pub their_channel_reserve_proportional_millionths: u32,
+	#[cfg(anchors)]
+	/// If set, we attempt to negotiate the `anchors_zero_fee_htlc_tx`option for outbound channels.
```

Comment:
I think here we can document LDK behavior in case of inbound channels not supporting `anchors_zero_fee_htlc_tx`, for now we don't provide an option to reject those ones and this feature likely coming in the future.